node/src/chain_spec.rs

```diff
@@ -178,24 +241,40 @@ fn testnet_genesis(
         },
         balances: BalancesConfig {
             // Configure endowed accounts with initial balance of 1 << 60.
-            balances: endowed_accounts.iter().cloned().map(|k| (k, 1 << 60)).collect(),
-        },
-        aura: AuraConfig {
-            authorities: initial_authorities.iter().map(|x| (x.0.clone())).collect(),
-        },
-        grandpa: GrandpaConfig {
-            authorities: initial_authorities.iter().map(|x| (x.1.clone(), 1)).collect(),
+            balances: endowed_accounts.iter().cloned().map(|k| (k, ENDOWMENT)).collect(),
         },
+        // This has been initialized from the session config
+        aura: Default::default(),
+        // This has been initialized from the session config
+        grandpa: Default::default(),
         sudo: SudoConfig {
             // Assign network admin rights.
             key: root_key,
         },
         utxo: UtxoConfig {
-            genesis_utxos: vec![genesis],
+            genesis_utxos,
+            // The # of validators set should also be the same here.
+            // Currently forcing only 1 INITIAL AUTHORITY (Alice), hence only 1 locked-genesis.
+            // This means only ALICE has staked her utxo.
+            locked_utxos: vec![locked_genesis],
+            extra_mlt_coins: 200_000  * MLT_UNIT,
+            initial_reward_amount: 100 * MLT_UNIT,
             _marker: Default::default(),
         },
         pp: PpConfig {
             _marker: Default::default(),
         },
-    }
+        session: SessionConfig {
+            keys: session_keys
+        },
+        staking: StakingConfig {
+            validator_count: initial_authorities.len() as u32 + 5u32, // provide 5 more slots
+            // The # of validators set should be the same number of locked_utxos specified in UtxoConfig.
+            minimum_validator_count: initial_authorities.len() as u32,
```

Comment:
What is the min num of validators?

Reply:
This can be changed; this was primarily for easy testing.
Currently the minimum count is set to 1, meaning there's only Alice.

Reply:
^ noted,